server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -84,15 +92,55 @@ func (p *Deployer) Deploy(ctx workflow.Context, requestedDeployment terraformWor
 		return nil, err
 	}
 
-	latestDeployment = requestedDeployment.BuildPersistableInfo()
-	if persistErr := p.persistLatestDeployment(ctx, latestDeployment); persistErr != nil {
-		logger.Error(ctx, "unable to persist deployment, proceeding with in-memory value", key.ErrKey, persistErr)
+	// log error and continue deploys if any of the post deploy task fails
+	if err := p.runPostDeployTasks(ctx, requestedDeployment); err != nil {
```

Comment:
Since we persist the deployment in the deployer, I decided to set revision for open PRs here as well instead of the worker for consistency since both of them are post deploy tasks and it should not block the further deploys. Lmk if you have better ideas on how to structure this change in the deploy workflow